download/config.go

```diff
@@ -14,6 +14,12 @@ import (
 const (
 	defaultMinDelaySeconds = int64(60)
 	defaultMaxDelaySeconds = int64(120)
+
+	// deltaBundleMode represents OPA supports delta bundle processing
```

Comment:
I might just be slow today, but I needed a moment to understand this 🙃 What would you think about

```suggestion
	// deltaBundleMode indicates that OPA supports delta bundle processing
```
or something? 🤔

Reply:
Updated